content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1694 @@
+
+Gets the global project that is registered to the engine. By default, this is read from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`. You can also specify a file path using `-i`.
+
+### Format
+
+```cmd
+get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+Reads the global project from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+```cmd
+o3de.bat get-global-project
+```
+
+Reads the global project from a specified file.
+```cmd
+o3de.bat get-global-project -i C:\Users\<USERNAME>\.o3de\Registry\my-custom.setreg
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-i INPUT_PATH, --input-path INPUT_PATH`**
+
+  A path to the input file that you want to read the  `/Amazon/AzCore/Bootstrap/project_path` key from. If not supplied, then this command uses the input file `C:/Users/USER_PATH/.o3de/Registry/bootstrap.setreg` instead.
+
+
+<!-------------------------------------------------------------->
+
+## `set-global-project`
+
+Sets the specified project as the engine's global project. By default this is set in `C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg`. You can also specify a file path using `-o`.
+
+### Format 
+
+```cmd
+set-global-project [-h] (-pp PROJECT_PATH | -pn PROJECT_NAME)
+                                  [-o OUTPUT_PATH] [-f]
+```
+
+### Usage
+
+Sets the specified project as the global project in `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+```cmd
+o3de.bat set-global-project -pp PROJECT_PATH
+```
+
+Sets the specified project as the global project in the specified path.
+
+```cmd
+o3de.bat set-global-project -pp PROJECT_PATH -o C:\Users\<USERNAME>\.o3de\Registry\my-custom.setreg
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-pp PROJECT_PATH, --project-path PROJECT_PATH`**
+
+  The path to the project.
+
+- **`-pn PROJECT_NAME, --project-name PROJECT_NAME`**
+
+  The name of the project. You can find the name in the project's `project.json` file.
+
+- **`-o OUTPUT_PATH, --output-path OUTPUT_PATH`**
+
+  A path to the output file that you want to write the `project_path` key to. If not supplied, then  this command uses the input file `C:/Users/USER_NAME/.o3de/Registry/bootstrap.setreg` instead.
+
+- **`-f, --force`**
+
+  Forces the project path to be set in the supplied setreg file. 
+
+
+<!-------------------------------------------------------------->
+
+## `create-template`
+
+Creates a template out of the specified source path.
+
+### Format
+
+``` cmd
+create-template [-h] -sp SOURCE_PATH [-tp TEMPLATE_PATH]
+                [-srp SOURCE_RESTRICTED_PATH | -srn SOURCE_RESTRICTED_NAME]
+                [-trp TEMPLATE_RESTRICTED_PATH | -trn TEMPLATE_RESTRICTED_NAME]
+                [-sn SOURCE_NAME]
+                [-srprp SOURCE_RESTRICTED_PLATFORM_RELATIVE_PATH]
+                [-trprp TEMPLATE_RESTRICTED_PLATFORM_RELATIVE_PATH]
+                [-kr] [-kl] [-r [REPLACE [REPLACE ...]]] [-f]
+```
+
+### Usage
```

Comment:
This command is complicated enough that it should have some additional examples. I know Derric has several more usage scenarios in his documentation on this Python script. If you don't have time to add these examples now, then I would suggest creating a GitHub issue for someone to do so later.

Reply:
https://github.com/o3de/o3de.org/issues/806